### PR TITLE
Cron job to sync cluster claim resource

### PR DESCRIPTION
**Pull request by indresh-28**

### CODE REVIEW

File: capten/agent/cmd/agent/main.go [hunk 1]
```diff
@@ -87,3 +103,21 @@ func configureDB() error {
...
+	}
+
+	_, jobErr := cronJob.AddJob(fmt.Sprintf(StrInterval, crontInterval), cron.NewChain(cron.SkipIfStillRunning(cron.DefaultLogger)).Then(fetch))
+	if jobErr != nil {
+		log.Errorf("Failed to add cronJob for sync clusterClaim: %v", jobErr)
+		return nil, jobErr
+	}
+
+	return cronJob, nil
+}
```
Comment (vramk23) [inline capten/agent/cmd/agent/main.go:123]: reuse the schduler.go in vault to add the jobs, its easier and clean to add jobs to scheduler

**Review by vramk23 — CHANGES_REQUESTED** [review]

File: capten/agent/cmd/agent/main.go (hunk 1 above)
Comment (indresh-28) [reply capten/agent/cmd/agent/main.go:123]: That's available under internal folder, can't be imported in other modules.

Comment (indresh-28) [reply capten/agent/cmd/agent/main.go:123]: That's available under internal folder, can't be imported in other modules.

Comment (vramk23) [reply capten/agent/cmd/agent/main.go:123]: reuse the code, not import the module

**Review by vramk23 — CHANGES_REQUESTED** [review]

**Pull request merged**